### ISSUE TITLE
Missing configurable AttributeConsumingService in Metadata Generator

### ISSUE THREAD

**Issue by impos89** [spring-projects/spring-security-saml#437]
Hi,
As I see, now it's not possible to configure md:AttributeConsumingService in MetadataGenerator for service provider.
Is it possible to do it in other way without overriding MetadataGenerator implementation ?

If it's missing, I can add support for that and share with pull request.

An example of expected result in sp metadata file:
` <md:AttributeConsumingService index="1">
            <md:ServiceName xml:lang="it">Nome del servizio</md:ServiceName>
            <md:ServiceDescription xml:lang="it">Descrizione del servizio</md:ServiceDescription>
            <md:RequestedAttribute Name="email" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:basic"/>
            <md:RequestedAttribute Name="name"
        </md:AttributeConsumingService>`

**Comment by fhanik** [comment]
Yes, a PR is greatly appreciated. Currently `md:AttributeConsumingService` is left out because the endpoint for that is not implemented.

**Comment by issilin** [comment]
@fhanik Hello! Is this task still relevant?
